,

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
@Sohalt this shouldn't matter since it's rewritten by `mkDerivation`

Here is the output of the script
```
#!/nix/store/j8vysakw78bpgngba32hfwwikqda9yx2-bash-4.4-p23/bin/bash
#
# usage example:
#   $ , yarn --help
# This finds a derivation providing a bin/yarn, and runs it with `nix run`.
# If there ar
```